src/components/OptionsPicker/OptionItem.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import type {StyleProp, ViewStyle} from 'react-native';
+import {View} from 'react-native';
+import Icon from '@components/Icon';
+import {PressableWithFeedback} from '@components/Pressable';
+import SelectCircle from '@components/SelectCircle';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import variables from '@styles/variables';
+import CONST from '@src/CONST';
+import type IconAsset from '@src/types/utils/IconAsset';
+
+type OptionItemProps = {
+    /** Text to be rendered */
+    title: string;
+
+    /** Icon to be displayed above the title */
+    icon: IconAsset;
+
+    /** Press handler */
+    onPress?: () => void;
+
+    /** Indicates whether the option is currently selected (active) */
+    isSelected?: boolean;
+
+    /** Indicates whether the option is disabled */
+    isDisabled?: boolean;
+
+    /** Optional style prop */
+    style?: StyleProp<ViewStyle>;
+};
+
+function OptionItem({title, icon, onPress, isSelected = false, isDisabled = false, style}: OptionItemProps) {
```

Comment:
```suggestion
function OptionItem({title, icon, onPress, isSelected, isDisabled, style}: OptionItemProps) {
```

I think `false` is unnecessary since it's a falsy value

Reply:
we can remove the default value from `isDisabled` right away, but if we were to remove it for `isSelected` too, we would have to add `?? false` to `<SelectCircle />` on **line 57**, do you think it's better?

Reply:
Let's keep `isSelected` then